src/renderer/i18n/ja.ts

```
@@ -31,6 +31,9 @@
       error: "設定の保存に失敗しました",
     },
   },
+  beat: {
+    badge: beat_badge,
```

ESLint

[error] 'beat_badge' is not defined.

(no-undef)